packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -168,6 +180,29 @@ export class OutgoingPayment
       updatedAt: json.updatedAt
     }
   }
```

Comment:
Can we not get rid of the `$formatJson` :point_up_2: ?
Ditto for incoming payments but not quotes
https://github.com/interledger/rafiki/blob/4df802bb059097546f21178505aff35d89a36757/packages/backend/src/open_payments/quote/service.ts#L296